crates/glaredb/tests/log_file_test.rs

```diff
@@ -0,0 +1,33 @@
+mod setup;
+
+use crate::setup::{make_cli, DEFAULT_TIMEOUT};
+
+#[test]
+fn test_default_log_file() {
+    let mut cmd = make_cli();
+    cmd.timeout(DEFAULT_TIMEOUT)
+        .arg("-q")
+        .arg("select 1;")
+        .assert()
+        .success();
+    let mut log_file = std::env::current_dir().expect("failed to retrieve current dir");
+    log_file.push("debug.log");
+    assert!(log_file.exists());
+}
```

Comment:
i'm not sure we want the default behavior to be write to a log file. IMO it should only write to log file if specified. @vrongmeal Thoughts on this? it looks like you opened up the original issue.

I'd personally prefer something like `glaredb --log-path debug.log` or `glaredb --log-file debug.log`, and if it's not specified, no logs are written to disk.